history-of-the-quakers/original/29-book-3-chapter-3.adoc

```diff
@@ -218,46 +245,45 @@ to which he replied, "`I love and honour all magistrates and rulers,
 who are for the punishment of evil doers, and the praise of them that do well.`"
 Then secretary Rawson called out, Edward Wharton come to the bar.
 
-Edward Wharton.
+_Edward Wharton._
```

Comment:
Maija, let's try this section again, but with `[.discourse-part]` labels -- i think it seems to fit pretty well with that markup.